services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -254,6 +235,13 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
                 dynamic_sidecar_endpoint
             )
         except BaseClientHTTPError:
+            if scheduler_data.dynamic_sidecar.were_containers_created:
+                # Containers disappeared after they were started.
+                # for now just mark as error and remove the sidecar
```

Comment:
ok this is nice, only remaining use-case is when a node is disconnected shortly. this will trigger this and we might end up in a double-save situation?